File: beginner/javascript/1864.js
var input = require('fs').readFileSync('stdin', 'utf8');

var linhas = input.split('\n');

var frase = "LIFE IS NOT A PROBLEM TO BE SOLVED";
var n = parseInt(linhas.shift())

var resultado = "";

for (var i = 0; i < n; i++)
    resultado += frase[i]

console.log(resultado);

File: beginner/java/1864.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);
        
        String frase = "LIFE IS NOT A PROBLEM TO BE SOLVED";
		int n = entrada.nextInt();

		for (int i = 0; i < n; i++)
			System.out.print(frase.charAt(i));

		System.out.println();
		
    }
}